mingw-w64-git/mingw-w64-git.mak

```diff
@@ -38,7 +40,7 @@ print-builtins:
 strip-all: strip
 	$(STRIP) $(STRIP_OPTS) \
 		contrib/credential/wincred/git-credential-wincred.exe \
-		cmd/git{,-gui,k}.exe compat-bash.exe git-{bash,cmd,wrapper}.exe
```

Comment:
Can potentially add this back; was trying some cross-compilation stuff before and it led to a segfault in the `strip-all` step. Will check this later

Reply:
It works when only `git-wrapper` is excluded from this line. Otherwise this leads to the following error:

```
/usr/src/mingw-packages/mingw-w64-git/src/git/../cv2pdb-strip  \
        contrib/credential/wincred/git-credential-wincred.exe \
        cmd/git{,-gui,k}.exe compat-bash.exe git-{bash,cmd,wrapper}.exe
/usr/src/mingw-packages/mingw-w64-git/src/git/../cv2pdb-strip: line 3: 32409 Segmentation fault      cv2pdb "$arg"
make: *** [../mingw-w64-git.mak:41: strip-all] Error 139
==> ERROR: A failure occurred in build().
    Aborting...
```